prompts/catalog.yaml

```diff
@@ -143,6 +143,13 @@ registry:
       - name: slack.team_id
     prompts: 0
     resources: {}
+    config:
```

Comment:
I think we should move the slack.team_id from secrets into config.  The team_id is mandatory but the channel_ids are optional. If you don't set channel_ids then you see all public channels.
Also, you can still /invite the bot to private channels and those channels will become available too.